backend/inspirehep/search_engines/tasks.py

```diff
@@ -54,4 +59,28 @@ def create_sitemap():
         for page_number in page_range
     ]
     index_content = render_template("sitemap/index.xml", urlset=index_items)
-    write_sitemap_page_content("", index_content)
+    upload_sitemap_content("", index_content)
+
+
+@shared_task(
+    ignore_results=False,
+    queue="rendertron",
+    acks_late=True,
+    retry_backoff=2,
+    retry_kwargs={"max_retries": 6},
+    autoretry_for=(
+        ConflictError,
+        ConnectionError,
+        ConnectionTimeout,
+        NotFoundError,
+        RequestError,
+        HTTPError,
+    ),
+)
+def render_records():
+    collections = get_rendered_records()
```

Comment:
I don't understand the strategy here. This is a single task, that will render all the records sequentially. Assuming a very conservative 3s/record for rendering+uploading and 1.6M records, this will take more than 55 days. I think it would make more sense to fire a rendering task, rendering and uploading a single record, on record update (and maybe skipping the rendering if the last render for that record is younger than N days).